src/core/components/TransactionStatus.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Grid, Paper, Typography } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ITransactionStatus } from '../interfaces';
+import dayjs from 'dayjs';
+import { HashPopover } from './HashPopover';
+
+interface Props {
+  status: ITransactionStatus;
+}
+
+export const TransactionStatus: React.FC<Props> = ({ status }) => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  return (
+    <>
+      <Typography fontWeight="bold">{t('status')}</Typography>
```

Comment:
Commented on the top-level too... but maybe "Details" is a more appropriate header here